recipes-devtools/opkg/opkg-keyrings_%.bbappend

```diff
@@ -15,6 +15,8 @@ SRC_URI += " \
 	file://nilrt-feed-2023.gpg \
 "
 
+PACKAGE_ADD_METADATA_IPK:opkg-keyrings = "DisplayName: Opkg-Keyrings\nUserVisible: yes\n"
```

Comment:
Move this declaration below the task definition (`do_install`) below, per [the styleguide](https://github.com/ni/meta-nilrt/blob/nilrt/master/scarthgap/docs/styleguide.md#layout).

Reply:
I have fixed this now.